src/components/LoginForm/LoginForm.tsx

```diff
@@ -46,17 +51,18 @@ function LoginForm({ changeStateFn }) {
     });
   };
 
-  const handleMouseDownPassword = (event) => {
+  const handleMouseDownPassword = (event: MouseEvent) => {
```

Comment:
What you probably need is `MouseEvent` defined by React: `import { MouseEvent, useContext, useState } from 'react';`.  You might consider importing it with an alias to prevent the name clash with the global variable `MouseEvent`.